completion/generate-completion.js

```diff
@@ -33,7 +33,7 @@ const commandsJson = JSON.parse(fs.readFileSync(commandsFilePath, 'utf8'));
 
 const mainCommands = [];
 const additionalCommands = [];
-for (const key of Object.keys(commandsJson.commands)) {
+for (const key of Object.keys(commandsJson.commands).sort()) {
```

Comment:
Object.keys won't guarantee always the same order of keys from a dict and this can cause intermitency on the completion.bash file which will lead to a CI break because some files were different than when pushed